fastruby/points
# Project Title inline edit

**Description:**

Currently, the only editable field of a project is the Title, to edit that we have to go to the Edit or Delete Project action, it requires a lot of clicks just for that and the Edit or Delete Project action has to deal with 2 tasks.

This PR fixes that by adding the option to edit the title field inline in the Show action of the projects. After editing the title it updates the navbar and sidebar when needed too.

With that change I was able to change the Edit or Delete Project button to be only a Delete Project button.

I also changed the layout a bit to fit all the content, if a project has a parent, the parent's title now appears smaller above the current project's title.

This inline-edit is only available in the show action.

https://user-images.githubusercontent.com/188464/154997698-48e3ae15-8a36-477b-9429-6f69ea1fe132.mp4

**How to QA:**

Go to the show action of different projects and click the title to edit it, try different options with that. Also check that it's not editable in other places using the same title.

Closes #149

___

I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/pull_request_template.md).
